Bugfixes -- camlaonam spawner / bases / etc rework

dijksterhuis opened the pull request:
ACAV Arsenal connected

ACAV heavy vehicles -- lowered

MPs cayuse -> slick

633rd moved the medical bird back slightly

633rd ch47 medical

ARVNR whitelisting for tanks

fix MP gate

mp slick

mp patrol vehicle

blackhorse 105s at base not behsdfkjhdsfsdfstanden

Blackhorse -- Removed Berch assets

blackhorse -- slicks too

mp_arsenal no longer safe zone

spread out acav vehicles

anzac new billboard

arvnr -- arvn specific spawners to fix whitelisting issues

Muskets -- OH6A spawn fixed

7thCav -- OH6A spawn fixed

teleporters -- fixed MP missing MACV and removed berch from list of options to TP to

Huckleberry87 committed:
Hotfix May 22

ACAV Arsenal connected

ACAV heavy vehicles -- lowered

MPs cayuse -> slick

633rd moved the medical bird back slightly

633rd ch47 medical

ARVNR whitelisting for tanks

fix MP gate

mp slick

mp patrol vehicle

blackhorse 105s at base not behsdfkjhdsfsdfstanden

Blackhorse -- Removed Berch assets

blackhorse -- slicks too

mp_arsenal no longer safe zone

spread out acav vehicles

anzac new billboard

arvnr -- arvn specific spawners to fix whitelisting issues

Muskets -- OH6A spawn fixed

7thCav -- OH6A spawn fixed

teleporters -- fixed MP missing MACV and removed berch from list of options to TP to

dijksterhuis commented:
testing

- ✅ ACAV Arsenal connected
- ✅ ACAV heavy vehicles -- lowered
- ✅ MPs cayuse -> slick
- ✅ 633rd moved the medical bird back slightly
- ✅ 633rd ch47 medical
- ✅ ARVNR whitelisting for tanks
- ✅ fix MP gate
- ✅ mp slick
- ✅ mp patrol vehicle
- ✅ blackhorse 105s at base not behsdfkjhdsfsdfstanden
- ✅ Blackhorse -- Removed Berch assets
- ✅ blackhorse -- slicks too
  - they've also got access to choktaws / seahorses now?
- ❌ mp_arsenal no longer safe zone
  - i'm gonna dig into this, should be working
- ✅ spread out acav vehicles
- ✅ anzac new billboard (after PR into your PR anyway)
- ✅ arvnr -- arvn specific spawners to fix whitelisting issues (wasn't the spawn issue iirc)
- ✅ Muskets -- OH6A spawn fixed
- ✅ 7thCav -- OH6A spawn fixed
- teleporters 
  - ✅ fixed MP missing MACV
  - ✅ removed berch from list of options to TP to
  
----

  todo (i'll PR onto this PR or you can fix if i've not done it -- read: fallen asleep)
  
  - there's an extra `}` character in the teleports config on like line 29
  - mp arsenal
  - comms (MP Base changes right after we've been saying everyone needs to wait 😁 )
  - anzac billboard paa

dijksterhuis committed:
Additional Fixes from DJ

- Arsenal markers on map
- teleporter typo fix
- add missing anzac billboard paa
- slight change to arsenals searching for `freefire_arsenal` in the anme of the sync'd object
- mp arsenal(s) no longer a safe zone

Huckleberry87 committed:
Merge pull request #4 from dijksterhuis/dj-pr-additional-fixes

Additional Fixes from DJ

dijksterhuis commented:
testing

✅ ACAV Arsenal connected
✅ ACAV heavy vehicles -- lowered
✅ MPs cayuse -> slick
✅ 633rd moved the medical bird back slightly
✅ 633rd ch47 medical
✅ ARVNR whitelisting for tanks
✅ fix MP gate
✅ mp slick
✅ mp patrol vehicle
✅ blackhorse 105s at base not behsdfkjhdsfsdfstanden
✅ Blackhorse -- Removed Berch assets
✅ blackhorse -- slicks too
✅  mp_arsenal no longer safe zone
✅ spread out acav vehicles
✅ anzac new billboard (after PR into your PR anyway)
✅ arvnr -- arvn specific spawners to fix whitelisting issues (wasn't the spawn issue iirc)
✅ Muskets -- OH6A spawn fixed
✅ 7thCav -- OH6A spawn fixed

teleporters
✅ fixed MP missing MACV
✅ removed berch from list of options to TP to

Huckleberry87 committed:
PR MR additional

Created a Carve out for TF to use the 113 mortar. (WL Complaint)

gave ARVN the WL for the AT 105mm. (should have by default)

Fixed RRR at Gia Lam. (Complaint)

cerebral-314 commented on maps/cam_lao_nam/mission.sqm:
```diff
@@ -211,104 +212,111 @@ class AddonsMetaData
 			url="https://www.arma3.com";
 		};
 		class Item19
+		{
```
Please unload ALL mods before working on the map.
Remove this dependency please.

cerebral-314 reviewed (changes requested):
Aside from the CBA dependency everything looks good.

Huckleberry87 committed:
PR Fix

Loaded with blank mods

Moved GH Spawn location -- should keep people from spawning on the roof.

cerebral-314 commented:
LGTM

dijksterhuis commented:
@cerebral-314 this was an earlier version of the one @wescottj98 merged prior so gonna close this.

(I messed up the push so had to create a new PR)
